docs/user_guide/installation.rst

```diff
@@ -42,9 +56,20 @@ The ``tqec`` package is a regular Python package that can be installed using ``p
 It is not (yet) available on the official Python Package Index PyPI, so you will have
 to manually provide the URL to install the package:
 
-.. code-block:: bash
+.. tab-set::
+
+    .. tab-item:: pip
+
+        .. code-block:: bash
+
+                python -m pip install git+https://github.com/tqec/tqec.git
+
+    .. tab-item:: uv
+
+        .. code-block:: bash
+
+                uv pip install git+https://github.com/tqec/tqec.git
```

Comment:
I have kept `uv pip install` in two places because this is the only option to install a non-editable version of tqec with uv. The default `uv sync` option to install a package is same as `pip install -e .`.